core/controllers/base_test.py

```diff
@@ -179,15 +179,6 @@ def setUp(self):
         # Update a config property that shows in all pages.
         self.signup(self.ADMIN_EMAIL, self.ADMIN_USERNAME)
         self.login(self.ADMIN_EMAIL, is_super_admin=True)
-        response = self.testapp.get('/admin')
-        csrf_token = self.get_csrf_token_from_response(response)
-        self.post_json('/adminhandler', {
```

Comment:
Oh, one thought. Do you want to get this PR in first? Note that it's the backend tests that are failing; the frontend ones ought to be fine since you haven't really changed the frontend. I'm not sure if you're able to run the backend tests on your machine, though?